library/CMakeLists.txt

```diff
@@ -125,6 +126,10 @@ if(LINK_WITH_PTHREAD)
     set(libs ${libs} pthread)
 endif()
 
+if(LINK_WITH_TRUSTED_STORAGE)
+    set(libs ${libs} trusted_storage)
+endif()
```

Comment:
Do we need to link with trusted_storage in Makefile builds as well? Seems odd to only add the possibility to link with trusted_stored for CMake.

Reply:
I've filed https://github.com/ARMmbed/mbed-crypto/issues/188 for this